app/models/mixins/ui_service_mixin.rb

```diff
@@ -0,0 +1,24 @@
+module UiServiceMixin
+  def icons
+    {
+      :ContainerReplicator => {:type => "glyph", :icon => "\uE624", :fontfamily => "PatternFlyIcons-webfont"}, # pficon-replicator
+      :ContainerGroup      => {:type => "glyph", :icon => "\uF1B3", :fontfamily => "FontAwesome"},             # fa-cubes
+      :ContainerNode       => {:type => "glyph", :icon => "\uE621", :fontfamily => "PatternFlyIcons-webfont"}, # pficon-container-node
+      :ContainerService    => {:type => "glyph", :icon => "\uE61E", :fontfamily => "PatternFlyIcons-webfont"}, # pficon-service
+      :ContainerRoute      => {:type => "glyph", :icon => "\uE625", :fontfamily => "PatternFlyIcons-webfont"}, # pficon-route
+      :Container           => {:type => "glyph", :icon => "\uF1B2", :fontfamily => "FontAwesome"},             # fa-cube
+      :Host                => {:type => "glyph", :icon => "\uE600", :fontfamily => "PatternFlyIcons-webfont"}, # pficon-screen
+      :Vm                  => {:type => "glyph", :icon => "\uE90f", :fontfamily => "PatternFlyIcons-webfont"}, # pficon-virtual-machine
```

Comment:
> Ah yes, decorator would be even better.  I didn't realize that topology was using decorators (or was planning to use them).

Actually decorators dont work well for topology\dashboard thats why i didnt use them :\

> Im guessing decorators are the way to go now but that means I have to iterate over all providers and count them by type instead having the DB do it.

we can go for something like `type.constantize.new.decorate.list_iconimage` but it just feels wrong...